central/scannerdefinitions/handler/handler.go

```diff
@@ -165,11 +165,6 @@ func writeErrorForFile(w http.ResponseWriter, err error, path string) {
 	httputil.WriteGRPCStyleErrorf(w, codes.Internal, "could not read file %s: %v", filepath.Base(path), err)
 }
 
-func serveFile(w http.ResponseWriter, r *http.Request, name string) {
-	log.Debugf("Serving vulnerability definitions from %s", filepath.Base(name))
-	http.ServeFile(w, r, name)
-}
-
```

Comment:
@RTann This is fine right?

Reply:
If it's unused I'd say so. @jvdm modified the scannerdefinitions handler recently, so he can probably confirm this, though